Cargo.lock

```diff
@@ -4,8 +4,12 @@ version = "0.1.0"
 dependencies = [
  "bson 0.1.2 (git+https://github.com/kyeah/bson-rs)",
  "byteorder 0.3.10 (registry+https://github.com/rust-lang/crates.io-index)",
+ "libc 0.1.8 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
I like the way the dependencies here are listed lexicographically. Maybe we should do the same when using crates? What does the rust style guide say about that?